components/compositing/compositor.rs

```diff
@@ -621,29 +622,29 @@ impl IOCompositor {
                 }
             },
 
-            CompositorMsg::WebDriverMouseButtonEvent(webview_id, action, button, x, y) => {
+            CompositorMsg::WebDriverMouseButtonEvent(webview_id, action, button, x, y, id) => {
```

Comment:
```suggestion
            CompositorMsg::WebDriverMouseButtonEvent(webview_id, action, button, x, y, msg_id) => {
```
Better clarity when there are multiple ids. Ditto below.